frontend/fontlist.lua

```diff
@@ -233,4 +233,14 @@ function FontList:getLocalizedFontName(file, index)
     return altname
 end
 
+function FontList:getFontArgFunc()
+    require("document/credocument"):engineInit()
```

Comment:
This will only be called when needed ? When ? Only when going to see the list in the menu ? Or when some other dispatcher bits are invoked ?

It's fine in here, as it uses other bits from this module.
May be name it `getFontArgFuncForDispatcher` to be clearer

Reply:
it is called once on `Dispatcher:init()`

Reply:
Mhhh, it's something I have tried to avoid, to not load cre until it is really needed (for people reading mostly PDFs where the little space gained by not loading it can be used for caching tiles).
Any way to keep calling it only when it's really needed?

Reply:
It will now only get called if you open a dispatcher submenu (like a specific gesture), but the downside is it now runs each time you open such a submenu (which granted should not be that often)

Reply:
Great! that's fine enough for me :)

> It will now only get called if you open a dispatcher submenu

Any submenu ? Or you need to go deep into "Reflowable documents >" and tap on "Set font >" ?

Reply:
any (`Tap Corner -> Bottom Right`)